android/src/main/java/com/horcrux/svg/RenderableShadowNode.java

```diff
@@ -204,6 +208,12 @@ public void draw(Canvas canvas, Paint paint, float opacity) {
                 mPath.setFillType(mFillRule);
             }
 
+            RectF r = new RectF();
+            mPath.computeBounds(r, true);
+            Matrix m = this.getSvgShadowNode().getReverseTransform();
```

Comment:
The canvas/context CTM should be possible to use instead of computing the reverse transform many times.

Reply:
I think it should be fine to use getMatrix, as we're creating the canvas, and it's never passed outside library code.